### ISSUE TITLE
Bug:

### ISSUE THREAD

**Issue by KillerStorm23** [ModernFlyouts-Community/ModernFlyouts#710]
Hi,

I actually installed modern flyouts a week ago. There is a bug which doesn't show the media playing. For example, when I open spotify, which song i play doesn't show. Or when i open Youtube, what i'm watching doesn't show. 

Please fix this bug as soon as possible. I really love modern flyouts and don't want it have these many bugs.
Can someone also tell me what to do to fix this issue?

Thank you.

**Comment by arnavbhate** [comment]
Is this the same as #702?

**Comment by KillerStorm23** [comment]
Yes, it is the same

**Comment by Cyberdroid1** [comment]
Can you please give us more info about your setup? Windows version, app version, and other details by following the proper bug template.

**Comment by KillerStorm23** [comment]
Yeah sure, the OS name is Windows home single language. Version 10.0.19043. The system model is HP ENVY notebook. The modern flyouts version which I installed is 0.9.3.0

**Comment by cmdshft** [comment]
Hi. I am also having this same exact issue. On latest Windows 10, the media flyout is the only thing not working. Neither iTunes or Spotify have this working with ModernFlyouts. Is there an ETA for a fix?

**Comment by Hiperfast25** [comment]
Hi. I haven't been able to fix the flyout. The flyouts don't appear for me, only the caps and lock numbers and whenever I try to turn off the audio flyout on the settings it crashes.

**Comment by arnavbhate** [comment]
I now have to repair this app from settings everyday as I have this error everyday.

**Comment by DustyWA** [comment]
Any update on this issue? 
I'm running Windows 10 Pro 21H2 and can't get the media controls to show at all. MFO version 0.9.3.0
I've tried Chrome, Edge and Spotify with no luck.
Volume, brightness and lock key flyouts all working perfectly.

Also a quick note, the app crashes whenever I try to disbale to the audio flyout.

**Comment by arnavbhate** [comment]
>Also a quick note, the app crashes whenever I try to disbale to the audio flyout.

This is #592 and will be fixed in the next update

>Any update on this issue?
>I'm running Windows 10 Pro 21H2 and can't get the media controls to show at all. MFO version 0.9.3.0
>I've tried Chrome, Edge and Spotify with no luck.

Have you tried resetting the app from settings? It temporarily fixes the issue.

**Comment by DustyWA** [comment]
I have tried resetting and reinstalling and the issue is still present.

**Comment by arnavbhate** [comment]
I have found that this problem occurs when I have the run at startup option enabled. If I disable it and create a task to manually start the app at startup, this problem does not occur.

**Comment by oveja10o01** [comment]
I'm who wrote the #909, what I should have to do now?

**Comment by oveja10o01** [comment]
As that this bugs will be solved in a future update of Modern Flyouts, thanks!
